python/vast/vast/convert.py

```diff
@@ -1,7 +1,11 @@
+import base64
 from collections import defaultdict
-from typing import Any, AsyncIterable
 from dataclasses import dataclass
+import datetime
+from decimal import Decimal
+from typing import Any, AsyncIterable
 
+import numpy as np
```

Comment:
Do we have to add `oldest-supported-numpy` to `pyproject.toml`?

Reply:
No, but they are a dependency of pandas that do have:
```
  "numpy>=1.20.3; python_version<'3.10'",
  "numpy>=1.21.0; python_version>='3.10'",
  "numpy>=1.23.2; python_version>='3.11'",
```

What is your concern exactly?

Reply:
We use `numpy` directly, so we should list it as a dep, no?

Reply:
ah yeah, I think you are right, that would be cleaner. Even though we are just using it to validate the output of a dep that uses it, I guess it is best practice to list all deps that are explicitly imported